resource/resource_graph.go

```diff
@@ -37,8 +40,8 @@ func NewGraph() *Graph {
 	}
 }
 
-// LastUpdatedTime returns the last logical clock time a resource updated.
```

Comment:
Given that I'm eventually adding a `time.Time` `lastReconfigured` field to graph nodes, `LastUpdatedTime` _seems_ like it would return a `time.Time`. I think it's more accurate to call this `LogicalClock` as it simply returns the value of that atomic.

Reply:
Great idea; switched to `CurrLogicalClockValue`.